openspec/specs/transient-error-notifications/spec.md

```
@@ -0,0 +1,64 @@
+## Requirements
+
+### Requirement: AgentIron SHALL notify users about visible transient action failures
+AgentIron SHALL use the global notification system to present failures from visible user-triggered actions when those failures would otherwise only be logged to the console.
+
+#### Scenario: Model switch fails
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)